cli/src/lib.rs

```diff
@@ -627,19 +636,22 @@ fn init(
     fs::write("Anchor.toml", toml)?;
 
     // Build virtual manifest.
-    fs::write("Cargo.toml", template::virtual_manifest())?;
+    fs::write("Cargo.toml", rust_template::virtual_manifest())?;
```

Comment:
No, it does not. However, we want solidity programs and rust programs to co-exists in the anchor workspace. So, it makes sense to have it in case some rust programs get added.

Reply:
No, it does not. I've removed it. Now if you create an solidity workspace with `anchor init -s` and then add a rust program with `anchor new`, then `Cargo.toml` gets added if does not yet exist.